envoy/registry/registry.h

```diff
@@ -334,22 +335,22 @@ template <class Base> class FactoryRegistry : public Logger::Loggable<Logger::Id
         continue;
       }
 
-      // Skip untyped factories.
-      std::string config_type = factory->configType();
-      if (config_type.empty()) {
-        continue;
-      }
+      for (const auto& config_type : factory->configTypes()) {
+        if (config_type.empty()) {
```

Comment:
nit: this is not needed anymore. It should probably assert it's non-empty.

Reply:
Yeah, an assert is enough here. Thanks.